.github/workflows/Dockerfile

```diff
@@ -0,0 +1,6 @@
+from espressopp/buildenv:ubuntu
+
+COPY espressopp/ /home/espressopp/espressopp
+RUN cmake -S espressopp -B espressopp-build -DCMAKE_BUILD_TYPE=Release -DEXTERNAL_MPI4PY=OFF
+RUN cmake --build espressopp-build -j2
```

Comment:
and shouldn't we install it too?

Reply:
I thought linking to the build folder is better as this also allows some development/rebuild in the docker container. If we use an installed version you need to reinstall.

Reply:
I added an install step to the CI runs.

Reply:
Install is currently broken. Will be a separate issue.